### PR TITLE
Ping/153 people page

### PR DESCRIPTION
**Pull request by pingchenchan**
## Description

1. Completed all features for the People page, except for updating volunteer status. 
2. The page design and 'Add People' component are adapted from Anthony's Inventory page and 'Add Item' component. 
3. The 'Add People' feature includes input validation and displays error messages if adding a new volunteer fails.
4. Updated on 11/21: finishing the three-dot button to change volunteer active status and display the PIN code.
 
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

_Link user story from projects.digitalaidseattle.org_
[PIT-153](https://das-ph-inventory-tracker.atlassian.net/browse/PIT-153)

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well as any relevant images for UI changes._

### CODE REVIEW

File: src/components/AddPeopleModal/AddPeopleModal.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
+import { useState } from 'react';
+import { Modal, Box, Typography, TextField, Button } from '@mui/material';
+
+type AddPeopleModalProps = {
```
Comment (bospoort) [inline src/components/AddPeopleModal/AddPeopleModal.tsx:4]: Suggestion to rename all People to Volunteer. We call it volunteer everywhere. And it is just volunteers.

**Review by bospoort — APPROVED** [review]
Looks good. Can you copy the code in ChatGPT and ask for suggestions? One of them is to split out components in separate files. My old React teachers started complaining about any file over 100 lines. :-)

File: src/components/AddPeopleModal/AddPeopleModal.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        const errorMessage = errorData?.error?.message || response.statusText;
+        throw new Error(errorMessage);
+      } else {
+        setSuccessMessage('Volunteer added successfully.');
+        fetchData();
+        resetInputsHandler();
+      }
+    } catch (error) {
+      console.error('Error posting to database:', error);
+      setErrorMessage(`Failed to add volunteer: ${(error as Error).message}`);
```
[the remark this answers is not in the document]
Comment (pingchenchan) [reply src/components/AddPeopleModal/AddPeopleModal.tsx:81]: image
The error message is already presented in this way, or do I need to change it to a snacknar?

**Review by jingtechy — APPROVED** [review]
It looks good to me, well done.

**Pull request merged**